oeps/oep-0007.rst

```diff
@@ -0,0 +1,526 @@
+===============================
+OEP-0007: Migrating to Python 3
+===============================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-7 <oep-0007>`                   |
++---------------+-------------------------------------------+
+| Title         | Migrating to Python 3                     |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-09-23                                |
++---------------+-------------------------------------------+
+| Author        | Cliff Dyer <cdyer@edx.org>                |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
```

Comment:
Don't forget to update this with the actual arbiter (not sure who it is for this OEP).